### ISSUE TITLE
Tiny ImageNet

### ISSUE THREAD

**Issue by vidit98** [rahafaljundi/Selfless-Sequential-Learning#1]
I really liked the idea of selfless sequential learning and have been trying it with different neural net architectures. Could you please provide the code that used to test on tiny ImageNet dataset as I am facing some issues with the same.
Thank you.

**Comment by rahafaljundi** [comment]
Thanks for the interest in our work. Can you please specify what issues you are facing? Do you need the hyperparameters used for these experiments?

**Comment by Balaji-Udayagiri** [comment]
I was trying to run Resnext architecture with self less sequential learning but I got memory error. I was using RTX 2080. It will be helpful if I can get the code/pretrained models  that was used for testing on tiny image net dataset. In the repo I found the code only for permuted MNIST dataset.

Secondly, I also wanted to know whether you have tested this on any big networks such as resnet50.
Thank you!

**Comment by vidit98** [comment]
I was trying to run Resnext architecture with self less sequential learning but I got a memory error. I was using RTX 2080. It will be helpful if I can get the code/pretrained models that was used for testing on tiny image net dataset. In the repo, I found the code only for permuted MNIST dataset.

Secondly, I also wanted to know whether you have tested the proposed algorithm on any big networks such as resnet50.
Thank you!

**Comment by rahafaljundi** [comment]
Actually, our loss is only implemented for the fully connected layers similar to dropout. Note that loss affects the parameters in the convolutional layers. However, extending the loss to convolutional layer needs some more work. 
For tiny ImagNet we use a variant of VGG Net explained in the paper. To run the code you need to pass the index of the layers where the loss needs to be applied. In vgg variant it is: in_layers=[[],['1','3']]. I will upload the model and sample code.

**Comment by vedanuj** [comment]
Hi @rahafaljundi How can I run SLNID on Tiny Imagenet? The file `SampleCode_Tiny_Imagenet.py` imports some libraries that are not present in the codebase. Can you please share the complete code? Thanks!

**Comment by rahafaljundi** [comment]
You can find TinyImagenet Demo in the code, please refer to README_TINYIMAGENET.md for more info.
